src/utils/validations.ts

```diff
@@ -12,26 +12,22 @@ interface CreatePost {
 }
 
 const isValidEmail = (email: string) => {
```

Comment:
pr이나 위키에 적어주셔도 감사하고 jsdoc을 활용해도 좋을 것 같습니다..!

```suggestion
/**
 * @description 이메일을 검증하는 로직입니다.
 */

const isValidEmail = (email: string) => {
  const emailPattern =/^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$/;

  return emailPattern.test(email.toLowerCase());
};
```

<img width="1440" alt="스크린샷 2023-09-21 오후 9 56 19" src="https://github.com/prgrms-fe-devcourse/FEDC4_HONKOK_JunilHwang/assets/79239852/f9c0046c-c4c6-43d4-9b9e-c7525825e3fe">

Reply:
해당 [PR](https://github.com/prgrms-fe-devcourse/FEDC4_HONKOK_JunilHwang/pull/190#issue-1906512120) 에 유효성과 관련된 내용 적어두었습니다!